pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -330,9 +320,20 @@ func orderBy(items []v3.OrderBy, tags []string) string {
 	for _, item := range items {
 		if item.ColumnName == constants.SigNozOrderByValue {
 			orderBy = append(orderBy, fmt.Sprintf("value %s", item.Order))
+			addedToGroupBy[item.ColumnName] = true
```

Comment:
Can you add a test for this? I think the above loop also adds a `#SIGNOZ_VALUE ASC/DESC` to the order by which will not exist in the select labels as `SIGNOZ_VALUE`.

Reply:
Didn't get you properly on this

Reply:
I think I misunderstood. Disregard this comment.